### PR TITLE
Loader gif

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -78,4 +78,4 @@ gulp.task('watch', function() {
 });
 
 gulp.task('styles', ['styles:dev', 'styles:prd','stylesAdmin']);
-gulp.task('default', ['styles', 'watch']);
+gulp.task('default', ['styles','compress']);
```
Comment (miltontony) [inline gulpfile.js]: Why the change here? I don't see how this is related to the loader? Just curious..

**Comment by nathanbegbie** [comment]
@Mitso Could we put a GIF in here of how the spinner behaves?

File: gulpfile.js (hunk 1 above)
Comment (Mitso) [reply gulpfile.js]: The JS is not included in the default tasks, will updated ['styles','compress','watch']

**Comment by Mitso** [comment]
Looks a bit tiresome

laoder

**Comment by Mitso** [comment]
@miltontony @nathanbegbie please review 

The ticket requires a spinner or progress bar. 
Decided to implement at a progress bar instead, it looks to be functioning better than a spinner however 2 issues with a page loader. 
1. it's implemented only after the DOM is loaded 
2. Is annoying to me at least - will potentially be a UX drop-off issue

I've throttled performance here:
laoder
